.github/workflows/CI.yml

```diff
@@ -73,13 +73,17 @@ jobs:
         # switching the user, as OpenSearch cluster can only be started as root/Administrator on linux-deb/linux-rpm/windows-zip.
         run: |
           chown -R 1000:1000 `pwd`
-          if lscpu  | grep -i avx2
+          if lscpu  | grep -i avx512f | grep -i avx512cd | grep -i avx512vl | grep -i avx512dq | grep -i avx512bw          
```

Comment:
```suggestion
          if lscpu  | grep -i avx512          
```

Reply:
that won't be accurate. We want all 5 flags to be present. The above check will return success, even if only 3/5 avx512 flags are found.
AVX512 has been incrementally improved over the years, so older systems or client machines may not have all of them.